applications/FluidDynamicsApplication/tests/cpp_tests/test_two_fluid_navier_stokes_element.cpp

```diff
@@ -386,6 +386,7 @@ namespace Kratos {
 			modelPart.AddNodalSolutionStepVariable(DISTANCE);
             modelPart.AddNodalSolutionStepVariable(NORMAL);
 			modelPart.AddNodalSolutionStepVariable(EXTERNAL_PRESSURE);
+			modelPart.AddNodalSolutionStepVariable(SLIP_LENGTH);

```

Comment:
this is not needed, you are using it non-historical

Reply:
Done.